### PR TITLE
Add basic 2-dof robot for examples and testing

### PR DESCRIPTION
**Pull request by eholum**
I've been poking around with PRM, and it's very helpful to have a low-DOF manipulator for sampling the configuration space. Could be helpful for future development since it makes testing that much easier/faster.

PRM!

https://github.com/sea-bass/pyroboplan/assets/7966037/86a1d609-02ea-4ee2-a534-b9c494d26c7a

### CODE REVIEW

File: src/pyroboplan/models/two_dof_description/two_dof.urdf [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!-- Simple 2-DOF robot arm for educational purposes -->
```
Comment (sea-bass) [inline src/pyroboplan/models/two_dof_description/two_dof.urdf:1]: Educational and testing!

**Review by sea-bass — COMMENTED** [review]
Good idea! Do you mind setting the two links to different colors like in this example?
https://github.com/sea-bass/pyroboplan/blob/main/examples/intro_pinocchio/intro_pinocchio_manual.py

That's all I got.

**Comment by eholum** [comment]
> Do you mind setting the two links to different colors like in this example?

Done! It's beautiful.

image

**Review by sea-bass — APPROVED** [review]
Color me impressed!

**Pull request merged**